packages/flutter/lib/src/rendering/error.dart

```diff
@@ -91,7 +91,7 @@ class RenderErrorBox extends RenderBox {
   /// See also:
   ///
   ///  * [minimumWidth], which controls how wide the box must be before the
-  //     horizontal padding is applied.
+  ///    horizontal padding is applied.
```

Comment:
There are a few more of these actually: https://github.com/flutter/flutter/pull/65693/files
If its not too much trouble could you apply them in this PR?

Reply:
Sure, I'll add them! Thanks for looking at this PR